cache/models/samples.js

```diff
@@ -485,7 +485,7 @@ module.exports = {
       return redisOps.getHashPromise(aspectType, aspectName);
     })
     .then((aspObj) => {
-      if (!aspObj) {
+      if (!aspObj || (aspObj.isPublished == 'false')) {
```

Comment:
probably do something like  `if (!aspObj || !aspObj.isPublished)`

Reply:
see comment above. the =='false' is to coerce the string into a boolean